docs/WIDGETS.md

```diff
@@ -51,3 +51,7 @@ For example, if we had software that was only compatible with Ubuntu 14.04 “Tr
 ```
 {% include os-matrix.html trusty=site.data.trusty.versions.summary.docker %}
 ```
+
+## Unused Image cleanup
+
+This is a Jekyll plugin that can detect what images aren't being used in our docs anymore and moves them to `assets/imgs/docs/_unused/`. You can find the source code for the script in `jekyll/_plugins/find_unused_images.rb`. The plugin hooks into Jekyll's build process and is turned off by default. 
```

Comment:
Is `assets/imgs/docs/_unused/` not in the final build output? If it is, what's the benefit of this?

Reply:
Hmm - I guess the path `assets/imgs/docs/_unused` is going to get bundled into the final output because it's in the assets path, eh?  I'm not sure where I should move them?

Or are you asking why keep the unused images at all? (if so, I thought they would be good to keep around for reference—even though they are in the version control history—at the very least, for those who aren't super familiar with git)

Reply:
err, actually, it seems that the build doe exclude based on the `_`underscore.

Reply:
If it isn't in the final site, than this makes sense to me. If it is, we might as well leave them where they were I think.

Also I think here, `assets/imgs/docs/_unused/` should be `assets/img/docs/_unused/`. Basically remove the 's'.

Reply:
Ok yep! we are going to leave `_unused` out of the final site. I have removed the `s`, good catch.

Reply:
cool